packages/nodes-base/nodes/ICalendar/createEvent.operation.ts

```diff
@@ -289,7 +289,7 @@ export async function execute(this: IExecuteFunctions, items: INodeExecutionData
 				end = start;
 			}
 
-			end = allDay ? moment(end).utc().add(1, 'day').format() : end;
+			end = allDay ? moment(end).utc().set({ hour: 23, minute: 59, second: 59 }).format() : end;
```

Comment:
this seems wrong 
![Screenshot 2024-10-10 at 16 09 37](https://github.com/user-attachments/assets/7e20190d-1e52-4908-8d98-82d5453f7b76)
![Screenshot 2024-10-10 at 16 10 10](https://github.com/user-attachments/assets/e2d1feae-6fbe-4c2e-9b22-1a8ee994e74a)